WebApplication/Backend/Repositorys/SurveyRepository.cs

```diff
@@ -0,0 +1,428 @@
+                statistics[0].AverageRating += Double.Parse(s.Question9);
+                statistics[0].increment(Int32.Parse(s.Question9));
+                statistics[1].AverageRating += Double.Parse(s.Question10);
+                statistics[1].increment(Int32.Parse(s.Question10));
+                statistics[1].AverageRating += Double.Parse(s.Question11);
+                statistics[1].increment(Int32.Parse(s.Question11));
+                statistics[1].AverageRating += Double.Parse(s.Question12);
+                statistics[1].increment(Int32.Parse(s.Question12));
+                statistics[1].AverageRating += Double.Parse(s.Question13);
+                statistics[1].increment(Int32.Parse(s.Question13));
+                statistics[2].AverageRating += Double.Parse(s.Question14);
+                statistics[2].increment(Int32.Parse(s.Question14));
+                statistics[2].AverageRating += Double.Parse(s.Question15);
+                statistics[2].increment(Int32.Parse(s.Question15));
+                statistics[2].AverageRating += Double.Parse(s.Question16);
+                statistics[2].increment(Int32.Parse(s.Question16));
+                statistics[2].AverageRating += Double.Parse(s.Question17);
+                statistics[2].increment(Int32.Parse(s.Question17));
+                statistics[2].AverageRating += Double.Parse(s.Question18);
+                statistics[2].increment(Int32.Parse(s.Question18));
+                statistics[3].AverageRating += Double.Parse(s.Question19);
+                statistics[3].increment(Int32.Parse(s.Question19));
+                statistics[3].AverageRating += Double.Parse(s.Question20);
+                statistics[3].increment(Int32.Parse(s.Question20));
+                statistics[4].AverageRating += Double.Parse(s.Question21);
+                statistics[4].increment(Int32.Parse(s.Question21));
+                statistics[4].AverageRating += Double.Parse(s.Question22);
+                statistics[4].increment(Int32.Parse(s.Question22));
+                statistics[4].AverageRating += Double.Parse(s.Question23);
+                statistics[4].increment(Int32.Parse(s.Question23));
+            }
+            for (int i = 0; i < 5; i++)
+            {
+                statistics[i].generatePercents();
+
+                if (i == 0)
+                    statistics[i].AverageRating = statistics[i].AverageRating / reports.Count/5;
+                else if (i == 1)
+                    statistics[i].AverageRating = statistics[i].AverageRating / reports.Count/4;
+                else if (i == 2)
+                    statistics[i].AverageRating = statistics[i].AverageRating / reports.Count/5;
+                else if (i == 3)
+                    statistics[i].AverageRating = statistics[i].AverageRating / reports.Count/2;
+                else if (i == 4)
+                    statistics[i].AverageRating = statistics[i].AverageRating / reports.Count/3;
+            }
```

Comment:
Why is this in a for loop if the numbers are fixed? Coundn't it just be:
 `statistics[0].AverageRating = statistics[0].AverageRating / reports.Count/5; `
and so on for each field?